test/datahike/test.cljc

```diff
@@ -37,6 +37,8 @@
     datahike.test.upsert
     ))
 
+(taoensso.timbre/set-level! :fatal)
```

Comment:
What levels should we use in Datahike?

Reply:
I thought it would make sense to set the level in the http-server for example. as a library I would not set anything, but I am open for other opinion. I just set it for testing, do you think this does not make sense?